core/urls.py

```diff
@@ -1,25 +1,75 @@
+import inspect
+
 from django.conf.urls import include
 from django.contrib import admin
+from django.db.models import fields
+from django.http import QueryDict
 from django.urls import path
+from django_filters.rest_framework import DjangoFilterBackend
 from djproxy.urls import generate_routes
+from rest_framework import parsers, viewsets
+from rest_framework.parsers import MultiPartParser
 from rest_framework.routers import SimpleRouter
 
+from . import serializers
 from . import views
 
 
+class MultiPartJsonParser(MultiPartParser):
+    def parse(self, stream, media_type=None, parser_context=None):
+        result = super().parse(stream, media_type=media_type, parser_context=parser_context)
+
+        data = {}
+        arrays = {}
+
+        model = None
+        if parser_context and 'view' in parser_context:
+            model = parser_context['view'].get_serializer_class().Meta.model
+        for key, value in result.data.items():
+            """
+                Handle ManytoMany field data, parses lists of comma-separated integers that might be quoted.
+                eg. "1,2"
+            """
+            if hasattr(model, key) and isinstance(
+                getattr(model, key), fields.related_descriptors.ManyToManyDescriptor
+            ):
+                print(value)
+                vals = value.replace('"', '')
+                arrays[key] = [x for x in vals.split(',')]
+            else:
+                data[key] = value
+
+        print(data)
+        print(arrays)
+        qdict = QueryDict('', mutable=True)
+        qdict.update(data)
+
+        for key in arrays:
```

Comment:
Rather than collecting information in `data` and `arrays`, this could build the new `QueryDict` directly.  That is, instead of have two loops, you could so something like:
```
        qdict = QueryDict('', mutable=True)
        for key, value in result.data.items():
            # Handle ManytoMany field data: parse lists of comma-separated
            # integers that might be quoted.  e.g. "1,2"
            if isinstance(
                getattr(model, key, None), fields.related_descriptors.ManyToManyDescriptor
            ):
                for val in value.split(','):
                    qdict.update({key, val.strip('"')})
            else:
                qdict.update({key, value})
```

Reply:
Done!